openedx/core/djangoapps/schedules/templates/schedules/edx_ace/recurringnudge_day3/email/subject.txt

```diff
@@ -1,3 +1,3 @@
 {% load i18n %}
 
-{% blocktrans %} {{course_name}} has started on edX {% endblocktrans %}
\ No newline at end of file
+{% blocktrans %}Keep learning in {{course_name}}!{% endblocktrans %}
```

Comment:
`{% trans %}`

Reply:
I think this one has to be a blocktrans since there is a variable substitution in it.